frontend/src/components/cronjob/Details.tsx

```diff
@@ -22,10 +34,15 @@ export default function CronJobDetails() {
   const [cronJob, cronJobError] = CronJob.useGet(name, namespace);
   const [isCronSuspended, setIsCronSuspended] = useState(false);
   const [isCheckingCronSuspendStatus, setIsCheckingCronSuspendStatus] = useState(true);
+  const [openJobDialog, setOpenJobDialog] = useState(false);
   const dispatch = useDispatch();
 
   function applyFunc(newItem: KubeObjectInterface) {
-    setIsCheckingCronSuspendStatus(true);
+    if (newItem.kind === 'CronJob') {
+      setIsCheckingCronSuspendStatus(true);
```

Comment:
Will this ever happen? I read here that you only run it with a Job kind.

Reply:
This is run for changing the suspend state too for CronJob